docs/toc.js

```diff
@@ -691,6 +691,29 @@ module.exports = {
             },
           ],
         },
+        {
+          title: 'Portable stories',
+          pathSegment: '',
+          type: 'menu',
+          children: [
+            {
+              pathSegment: 'portable-stories-jest',
```

Comment:
@yannbf — Maybe we don't need `portable-stories` in the URL? `storybook.js.org/docs/jest` vs. `storybook.js.org/docs/portable-stories-jest`. Wdyt?

Reply:
That's a good question, I guess as long as SEO is good for these cases:
- Storybook tests jest
- Storybook portable stories
- Storybook unit tests

it should be good, but I'm no expert.

@winkerVSbecks could you chime in? For context, we are writing docs for the `composeStories` functions which we are trying to surface as "portable stories". There will be multiple pages, one for each use case (Jest, Vitest, Playwright CT). Thanks <3

Reply:
Will this become the primary integration with those tools going forward? If so, then I would go with `storybook.js.org/docs/jest`, etc. If this is one of multiple ways you can use Storybook with Jest/Vitest/etc. then I would use `portable-stories-jest`, etc.

Reply:
I forgot about the `api` directory in the route, so the best we could do is `storybook.js.org/docs/api/jest`. I think that 1) is not that much better than `storybook.js.org/docs/api/portable-stories-jest`, and 2) warrants a proper `storybook.js.org/jest` page, if we deem it important (in a future effort).